wire-ios-utilities/Source/DeveloperFlag.swift

```diff
@@ -23,33 +23,17 @@ public enum DeveloperFlag: String, CaseIterable {
     public static var storage = UserDefaults.standard
 
     case showCreateMLSGroupToggle
-    case breakMyNotifications
     case nseV2
-    case nseDebugging
-    case nseDebugEntryPoint
-    case useDevelopmentBackendAPI
     case deprecatedCallingUI
 
     public var description: String {
         switch self {
         case .showCreateMLSGroupToggle:
             return "Turn on to show the MLS toggle when creating a new group."
 
-        case .breakMyNotifications:
-            return "Turn on to get your app in a state where it no longer receives notifications."
-
         case .nseV2:
             return "Turn on to use the new implementation of the notification service extension."
 
-        case .nseDebugging:
-            return "Turn on to make the notification service extension (NSE) display debug notifications."
-
-        case .nseDebugEntryPoint:
```

Comment:
you delete this but not the debug content https://github.com/wireapp/wire-ios-mono/pull/130/files#diff-6486c5aabc91c37d10e4f90c897c5b9177030c76242a5e7f10ab310892bc9e1bR208, normal?

Reply:
I missed that, now it's deleted.